examples/flutter_gallery/android/.gitignore

```diff
@@ -8,6 +8,6 @@
 /captures
 GeneratedPluginRegistrant.java
 
-/gradle
+gradle-wrapper.jar
```

Comment:
We really shouldn't exclude the gradle wrapper files (including the .jar), if we want this to serve as an example of what an app should look like.